.github/workflows/ci.yml

```diff
@@ -34,9 +34,8 @@ jobs:
       fail-fast: true
       matrix:
         include:
+          - ver: 8.0
           - ver: 7.7.0
-          - ver: 7.6
```

Comment:
why remove for the sake of removing?  up to you of course

Reply:
you are not pushing people forward with this change, you only remove information for us - we won't know when things break. So I would recommend reverting the removal

Reply:
we will know at what time something broke, and make an informed decision that X broke something. Since its free for us, i don't see why not keep it

Reply:
Because the path is set: if something breaks on an unsupported release, I'm taking it out of the CI, there's no informed decision, just a predetermined action. Hence there's no reason to not take it out out now.

But if you want to revert that change, go ahead, as long as it doesn't break, it's fine.